app/parsers/bulkrax/bagit_parser.rb

```diff
@@ -33,10 +33,14 @@ def import_fields
     # Assume a single metadata record per path
     # Create an Array of all metadata records, one per file
     def records(_opts = {})
-      raise 'No metadata files were found' if metadata_paths.blank?
-      @records ||= metadata_paths.map do |path|
+      raise 'No BagIt records were found' if bags.blank?
+      @records ||= bags.map do |bag|
+        path = metadata_path(bag)
+        raise 'No metadata files were found' if path.blank?
         data = entry_class.read_data(path)
-        entry_class.data_for_entry(data, path)
+        data = entry_class.data_for_entry(data, path)
```

Comment:
`data` assigned twice

Reply:
It's being re-assigned once it's being used so it's fine, but maybe a different variable name would be sensible here?

Reply:
I kind of want the old value to be garbage collected as it can be significant